Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/TaskContainerFragment.java

```diff
@@ -126,22 +150,81 @@ private void handleAssetSelecting(Intent data, AssetType assetType) {
             Long assetId = assetRepository
                     .create(AssetType.getTypeByExtension(assetName), assetName);
             setAssetValue(assetType, assetName, assetId);
-
         } catch (IOException e) {
-            String pickingFileError = getResources().getString(R.string.picking_file_error);
-            Toast errorToast = Toast.makeText(context, pickingFileError, Toast.LENGTH_LONG);
-            errorToast.show();
+            showToastMessage(R.string.picking_file_error);
         }
     }
 
     private void setAssetValue(AssetType assetType, String assetName, Long assetId) {
+
```

Comment:
Mikolay, could you please help me with regex for pattern "name_5986.mp3" if you give me the answer, I'll trim the name in one line: 
assetName = assetName.replace("regExp", ""); Will it work ?